work/flow.rst

```diff
@@ -474,11 +474,22 @@ We **always use non-fast forward merges** so that the merge point is marked in G
 
 .. code-block:: bash
 
+   # Prepare to merge
+   git checkout master
+   git pull  # Sanity check; if master is up-to-date, skip ahead to "Merge" below
+   git checkout tickets/DM-NNNN
+   git rebase master
+   git push --force  # so that the commits on the remote branch match the commits merged*
+   # Merge
```

Comment:
Delete this comment.

Now add the paragraph + list starting with "We force push..." here.

Then introduce a new code block:

```
Next, merge to ``master`` and push.
We **always use non-fast forward merges** so that the merge point is marked in Git history, with the merge commit containing the ticket number:

.. code-block:: bash

   git checkout master
   git merge --no-ff tickets/DM-NNNN
   git push
```

Reply:
This means you can delete the identical paragraph, `We **always use ...` from before the two code blocks.